src/main/java/org/jboss/pnc/deliverablesanalyzer/model/FinderResultCreator.java

```diff
@@ -60,6 +64,52 @@ public static FinderResult createFinderResult(String id, URL url, Map<BuildSyste
                 .build();
     }
 
+    private static void setLicenseInformation(Artifact.ArtifactBuilder builder, KojiLocalArchive localArchive) {
+        Set<LicenseInfo> licenses = Optional.ofNullable(localArchive.getLicenses())
+                .orElse(Collections.emptySet())
+                .stream()
+                .map(FinderResultCreator::toDTO)
+                .collect(Collectors.toSet());
```

Comment:
Actually, `getLicenses()` should never return `null`. The default is initialized to an empty `TreeSet`, so you should be able to just simplify to `localArchive.getLicenses().stream()`.